messagebird/signed_request.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
+import hmac
+import base64
+import time
+from collections import OrderedDict
+
+try:
+    from urllib.parse import urlencode
+except ImportError:
+    from urllib import urlencode
+
+
+class SignedRequest:
+
+    def __init__(self, requestSignature, requestTimestamp, requestBody, requestParameters):
+        self._requestSignature = requestSignature
+        self._requestTimestamp = str(requestTimestamp)
+        self._requestBody = requestBody
+        self._requestParameters = requestParameters
+
+    def verify(self, signing_key):
+        payload = self._build_payload()
+        expected_signature = base64.b64decode(self._requestSignature)
+        calculated_signature = hmac.new(signing_key.encode('latin-1'), payload.encode('latin-1'),
```

Comment:
why `latin-1` ?

Reply:
because it's the way if should be encoded in order to generate the right signature, I tested utf-8 and didn't work with the same signature as the other SDKs